### ISSUE TITLE
Bad parameter names

### ISSUE THREAD

**Issue by jmathew** [Azure/azure-docs-cli-python#577]
The parameter for logging in with the service principal uses --user-name. The value passed into that parameter is actually the application Id. 

This is super confusing. I understand re-using parameters that already exist for simplicity, but in this case it actively lead to me fumbling around searching for my service principal's non-existant username for a long while. 

In hindsight, I can see that the docs are clear about using the appId. But my starting situation only half matched this article's starting state. In my case I already had a service principal that I was looking at in the portal. So logically I skipped the sections about creating the SP and getting details about it. And went straight into the login portion. And that part doesn't directly correlate the '-u' flag to the appId. I have to figure that out by process of elimination. 

Long story short: why not '--service-principal-name'?

Also, along the same lines; the portal lists what constitutes the 'password' as a 'Key'. The terminology should be consistent across management tools.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: b11fa31b-582b-7d4f-9a64-60b29da29565
* Version Independent ID: 03a47d06-5fbf-4764-0882-99cd8bd41110
* [Content](https://docs.microsoft.com/en-us/cli/azure/create-an-azure-service-principal-azure-cli?view=azure-cli-latest)
* [Content Source](https://github.com/Azure/azure-docs-cli-python/blob/live/latest/docs-ref-conceptual/create-an-azure-service-principal-azure-cli.md)

**Comment by micahmckittrick** [comment]
Thanks for the detailed information about your issue. We have requested that the content team reply to you to better address this.

**Comment by tfitzmac** [comment]
It seems like this issue is focused on the name of the parameter in product, not the documentation. The documentation is just showing the name of the parameter in Azure CLI.

@micahmckittrick - can this issue get reassigned to the Azure CLI product team?

**Comment by micahmckittrick** [comment]
@tfitzmac yes can do.

**Comment by tfitzmac** [comment]
@jmathew - please [submit this as a request](https://feedback.azure.com/forums/216665-scripting-and-command-line-tools) to the Azure CLI team. 

#please-close

**Comment by sptramer** [comment]
Closing as a feature request.
